src/views/TabView/TabBarBottom.js

```diff
@@ -51,11 +53,16 @@ type Props = {
   tabStyle?: ViewStyleProp,
   showIcon?: boolean,
   isLandscape: boolean,
+  initialLayout?: Layout,
+  layout: Layout,
+  enableHorizontalTabs: boolean,
```

Comment:
Props are usually named as `scrollEnabled` etc. Also, I'm not sure `horizontalTabs` is the correct name since it's the labels inside the tabs which are horizontal, and to me horizontal or vertical tabs feel like the tabs themselves are arranged in that direction. I'd just name this prop something like `adapative` or `responsive` unless you have a better name suggestion.

Reply:
I see why horizontalTabs could be confusing, I also feel that adaptive/responsive is too generic and you may want those options to do something else (non iOS 11 specific) in the future.  That being said, I can't think of a better name.